plays/aws-win-env/README.md

```
@@ -0,0 +1,52 @@
+# aws-win-env
+
+This playbook is intended to provide an easily deployable Windows instance in AWS for testing DLS in an environment that mimics enterprise environment limitations such as:
+- Non-Admin user
+- HTTP/2 disabled
+- WDAC Profiles and AppLocker (TBD)
+
+## Prerequisites
+If on Fedora:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)